src/hooks/useInterval.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+
+export default function useInterval(callback: () => void, delay?: number) {
+  const savedCallback = React.useRef<() => void>();
+
+  // Remember the latest callback.
+  React.useEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+
+  // Set up the interval.
+  React.useEffect(() => {
+    function tick() {
+      savedCallback.current && savedCallback.current();
+    }
+    if (delay !== null) {
+      const id = setInterval(tick, delay);
+      return () => clearInterval(id);
+    }
+  }, [delay]);
+}
```

Comment:
why do we need that?

Reply:
We don't need, this was an old function that I forgot to delete